source/adapters/cuda/CMakeLists.txt

```diff
@@ -67,6 +67,12 @@ else()
   )
 endif()
 
+if(UMF_ENABLE_POOL_TRACKING)
+  target_compile_definitions("ur_adapter_cuda" PRIVATE UMF_ENABLE_POOL_TRACKING)
```

Comment:
Should we add this option in the readme?

Reply:
as far as I know the UMF team are reworking stuff so that this option isn't needed, hopefully its days are numbered

Reply:
This is already an option in UR. It was just not being used in the adapter